src/main/scala/com/github/unchama/seichiassist/util/BreakUtil.scala

```diff
@@ -276,12 +278,14 @@ object BreakUtil {
    * world 内での整地量倍率を計算する。
    * TODO: これはビジネスロジックである。breakcountシステムによって管理されるべき。
    */
-  def blockCountWeight(world: World): Double = {
+  def blockCountWeight[F: JavaTime](world: World): F[Double] = {
```

Comment:
```suggestion
  def blockCountWeight[F[_]: JavaTime](world: World): F[Double] = {
```

`JavaTime`は`JavaTime[T[_]]`なので一階の高階型である必要があります

Reply:
はえ...

Reply:
soreha.soになってしまった